src/views/assets/Asset/AssetDetail/Detail.vue

```diff
@@ -183,8 +183,8 @@ export default {
         url: `/api/v1/assets/assets/${this.object.id}/`,
         object: this.object,
         fields: [
-          'id', 'name', 'category', 'type',
-          'address', 'platform', 'protocols', 'zone',
+          'id', 'name', 'category', 'type', 'address',
+          'platform', 'protocols', 'zone', 'directory_services',
           'is_active', 'date_created', 'date_updated',
           'created_by', 'comment'
         ],
```

Comment:
There aren't any noticeable code changes that could lead to issues, irregularities, or optimization improvements. The snippet appears mostly untouched from the original version shared until now, maintaining its functionality without introducing new bugs or inefficiencies that are not already apparent based on current knowledge.

Would you like further analysis? Or do you need assistance with another technical problem related to this code?